core/src/com/unciv/logic/automation/civilization/NextTurnAutomation.kt

```diff
@@ -321,6 +320,98 @@ object NextTurnAutomation {
                 city.cityConstructions.purchaseConstruction(construction.name, 0, true)
             }
         }
+
+        maybeBuyCityTiles(civInfo)
+    }
+
+    private fun maybeBuyCityTiles(civInfo: Civilization) {
+        if (civInfo.gold <= 0)
+            return
+
+        val highlyDesirableTiles: SortedMap<Tile, MutableSet<City>> = TreeMap(
+            compareByDescending<Tile?> { it?.naturalWonder != null }
+                .thenByDescending { it?.resource != null && it.tileResource.resourceType == ResourceType.Luxury }
```

Comment:
Luxury, that you don't own yet... So I guess this is more common in the early game and then I'd say, hell yes it's more important. In the later game this will rarely happen anyways.